File: python/opencv/opencv-old/image/plus.py
# -*- coding: utf-8 -*-
import cv2

def main():
    # 2枚の画像をグレースケールで取得
    im1 = cv2.imread("test1.png",0)
    im2 = cv2.imread("test2.png",0)
    # 画像データを重ねあわせ
    im = im1 + im2
    cv2.imshow("test",im)
    cv2.waitKey(0)
    cv2.destroyAllWindows()    # ウィンドウ破棄
    
if __name__ == '__main__':
    main()

File: cpp/opencv/image/Plus.cpp
#include "opencv2/opencv.hpp"

using namespace cv;

int main(int argc, char* argv[])
{
	Mat im1 = imread("test1.png");	// 入力画像1の取得
	Mat im2 = imread("test2.png");	// 入力画像2の取得
	Mat im = im1 + im2;				// 入力画像1と2の加算
	imshow("TEST", im);				// 結果を表示
	waitKey();						// キー入力待機
	return 0;
}
